### ISSUE TITLE
Please update `pubspec.yaml`

### ISSUE THREAD

**Issue by anthonymoretti** [kyle-seongwoo-jun/flutter_timezone_to_country#3]
Hi

Is it possible to update `pubspec.yaml` and use `2024b`?

From:
```
  timezone: ^0.9.3
  flutter_timezone: ^1.0.4
```

To:
```
  timezone: ^0.10.0
  flutter_timezone: ^4.0.0
```

Thank you!

**Comment by kyle-seongwoo-jun** [comment]
@anthonymoretti hi, I just released 3.0.0 version. Thanks :)

**Comment by anthonymoretti** [comment]
Thank you very much :)
